firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1270,13 +1270,24 @@ class BrowserViewController: UIViewController,
             state = isLoading ? .stop : .reload
         }
 
-        navigationToolbar.updateMiddleButtonState(state)
+        handleMiddleButtonState(state)
         if !toolbar.isHidden {
             urlBar.locationView.reloadButton.reloadButtonState = isLoading ? .stop : .reload
         }
         currentMiddleButtonState = state
     }
 
+    private func handleMiddleButtonState(_ state: MiddleButtonState) {
+        // TODO: Use REDUX to show felt deletion https://mozilla-hub.atlassian.net/browse/FXIOS-7994
```

Comment:
@adudenamedruby I created a redux ticket and assume we wanna have a `feltDeletionState` of some sort?